rpyc/utils/teleportation.py

```diff
@@ -117,7 +117,10 @@ def _import_codetup(codetup):
 
 def import_function(functup):
     name, modname, defaults, codetup = functup
-    mod = __import__(modname, None, None, "*")
+    try:
```

Comment:
This change seems unrelated to the PR topic. Also, I think it makes generally no sense to use  `__main__` as fallback here (or does it?) and letting the import fail will result in better error messages. I will probably cherry-pick around this commit.